Dockerfile

```diff
@@ -1,8 +1,57 @@
-FROM geonode/django
-MAINTAINER GeoNode development team
+FROM python:2.7.9
```

Comment:
You mean `geonode/django` is a base image, also used in https://github.com/GeoNode/geonode-project/blob/docker/Dockerfile#L1. We were not aware of it.

The `terranodo/django` image is indeed built from https://github.com/GeoNode/django-docker but this lacks readability, not to mention the `geonode` image tag.

The `geonode-project` duplicates code... where should we contribute ?
We're a bit lost ;-)